tutordistro/commands/enable_private_packages.py

```diff
@@ -26,6 +26,9 @@ def enable_private_packages():
     """
     directory = subprocess.check_output("tutor config printroot", shell=True).\
         decode("utf-8").strip()
+    plugin_directory = directory + "/tutor-contrib-edunext-distro/tutordistro/plugin.py"
+    print('hi')
+    print(plugin_directory)
```

Comment:
Probably we want to remove these lines 🙈

Reply:
You're right, my bad!